qa/common/util.sh

```diff
@@ -261,6 +261,7 @@ function run_server_leakcheck () {
       echo "=== Running LD_PRELOAD=$SERVER_LD_PRELOAD $SERVER $SERVER_ARGS"
     fi
 
+    TRITONSERVER_DISABLE_BACKEND_UNLOAD=1
```

Comment:
You have to put it on the same line as the executable and then the envvar setting applies only to that executable (like the LD_PRELOAD in the following line). That way this setting won't make it into the shell env and perhaps impact other runs of tritonserver. You just leave a space:

TRITONSERVER_DISABLE_BACKEND_UNLOAD=1 LD_PRELOAD=$SERVER_LD_PRELOAD $LEAKCHECK ...

Reply:
Ah okay got it, I've been trying multiple ways of doing this, and some mess up the SERVER_PID=$! line but I forgot about this way. Thanks!